src/libsystemd/sd-journal/journal-verify.c

```diff
@@ -824,6 +824,40 @@ static int verify_entry_array(
         return 0;
 }
 
+static int verify_hash_table(
+                Object *o, uint64_t p, uint64_t *n_hash_tables, uint64_t header_offset, uint64_t header_size) {
```

Comment:
doesn't matter, but maybe add assert() on the stuff you are about to dereference unconditionally below